.github/workflows/validation.yml

```diff
@@ -0,0 +1,62 @@
+name: Validation
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+env:
+  BUILD_TYPE: Debug
+  FRAMEWORK_PATH: /__w/restG4/restG4/framework
+
+defaults:
+  run:
+    shell: bash -ieo pipefail {0}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: lobis/root-geant4-garfieldpp:cxx14_ROOTv6-25-01_Geant4v10.4.3
```

Comment:
I am not an expert on this topic, but shall the image point to lobis?

Reply:
Thats the image we are using for validation, created by @lobis. Why that comment? What do you mean? To have a docker account named `rest-for-physics`?

Reply:
Yes, that would be better I guess. I guess if we do that, we should host the docker image via github packages, which is the github way of doing this.